tf_files/aws/modules/common-logging/lambda_function.py

```diff
@@ -251,6 +251,32 @@ def nice_it(r_data):
     return individuals
     
 
+##
+#
+# send_to_logDNA function that invokes amother lambda function, in this case specifically, our logDNA lambda function
+#
+# @var payload String log stream to send
+# 
+# @return null
+#
+##
+
+def send_to_logDNA(payload):
+
+
+    try:
+        # if there is no threshold, let's not even check
+        if os.environ.get('log_dna_function') is not None:
+            log_dna_function = os.environ.get('log_dna_function')
```

Comment:
Can we add a check here that the `log_dna_function` looks like an `^arn:`, so we can pass an empty string or whatever through as an environment variable?

Reply:
validation added in latest commit